src/actors/snapshots/symbolicator__actors__symbolication__tests__add_bucket-2.snap

```diff
@@ -31,6 +31,7 @@ modules:
     candidates:
       - source: local
         location: 502F/C0A5/1EC1/3E47/9998/684FA139DCA7
+        uri: "http://localhost:<port>/download/502F/C0A5/1EC1/3E47/9998/684FA139DCA7"
```

Comment:
Q: Can we just make this the `location`?

Reply:
yeah, i imagine the UI doesn't want both.  Do we:
1. store the value of `uri` above in `location`
2. leave `uri` as above and remove `location` (will break current UI)

just using location gives us a little more leeway as it does not imply a URI of any sort (and there's the weird `noent://` case to deal with).

Reply:
I'd go with Option 1 since otherwise location and URI seem like duplicate information. Agreed, `noent://*` is strange. In which case would it appear?